src/lib/containers/table/TopLevelControls.tsx

```diff
@@ -156,18 +149,31 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
       <div>
         <div className="TopLevelControls__querycount">
           {name && (
-            <>
+            <Typography variant="sectionTitle" role="heading">
+              {name}{' '}
               {!hideQueryCount && (
-                <TotalQueryResults
-                  frontText={''}
-                  endText={hasFacetsOrFilters ? 'Filtered By' : ''}
-                  showNotch={showNotch}
+                <QueryCount
+                  sql={getInitQueryRequest().query.sql}
+                  parens={true}
                 />
               )}
-              {!hideQueryCount && entity && (
-                <MissingQueryResultsWarning entity={entity} />
-              )}
```

Comment:
Where is MissingQueryResultsWarning going to go? Didn't see where it was relocated, if it was

Reply:
Missed that in the revert (which is why the build failed too!).  Brought it back